crates/math/src/fold.rs

```diff
@@ -260,7 +260,8 @@ where
 {
 	if log_evals_size < log_query_size {
 		bail!(Error::IncorrectQuerySize {
-			expected: log_evals_size
+			expected: log_evals_size,
+			actual: log_query_size,
```

Comment:
I had some doubts on this one: shouldn't those be swapped?

Reply:
I think it's good this way. Means that `log_query_size` is expected to be `<= log_evals_size`.